_maps/skyrat/automapper/automapper_config.toml

```diff
@@ -22,8 +22,8 @@ trait_name = "CentCom"
 [templates.centcom_shuttle_dock]
 map_files = ["centcom_shuttle_dock.dmm"]
 directory = "_maps/skyrat/automapper/templates/centcom/"
-required_map = "builtin"
-coordinates = [206, 85, 1]
+required_map = "CentCom_skyrat_z2.dmm" ## BUBBERSTATION EDIT - MOVES THE SHUTTLE TO THE CENTCOM DOCK
+coordinates = [51, 167, 2]			   ## BUBBERSTATION EDIT
```

Comment:
Atomize this into another PR

Reply:
I didn't do this part, it was just pulling master